### ISSUE TITLE
Uncorrect warning: Property "compile" in ts task is possibly in the wrong place and will be ignored.

### ISSUE THREAD

**Issue by 0815fox** [TypeStrong/grunt-ts#364]
Having the following config in Gruntfile:

```
ts: {
  compile : {
    tsconfig: true
  }
}
```
pops out following warning:
```
Property "compile" in ts task is possibly in the wrong place and will be ignored.  It is expected on the options object.
```
This is the same config like in [this chapter](https://github.com/TypeStrong/grunt-ts#tsconfig) of Readme.md - the only difference is, that I named my target "compile" - so the compile is not meant as the compile-option on the options object, rather it is meant to be the name of the ts task.

**Comment by nycdotnet** [comment]
Fair enough.  I agree this is a false positive.  It still works, right?

**Comment by 0815fox** [comment]
Yeah it works, just confused me in the beginning. So it is not critical at all.

**Comment by nycdotnet** [comment]
This should be fixed in `grunt-ts@6.0.0-beta.2`.  Please confirm.  Thank you for the report.

**Comment by nycdotnet** [comment]
Hi @0815fox - I believe this is fixed, but please reopen if it is not.  Thank you.
